server/workers/countCrimesPerStreet.js

```diff
@@ -0,0 +1,48 @@
+var request = require('request');
+var db = require('../models/rating.js');
+var crime = require('../models/crime');
+
+var updateCrimeCounter = function(lat, lng) {
+  var url =  `http://locationiq.org/v1/reverse.php?format=json&key=6bc67043075bb2c0f0b3&lat=${lat}&lon=${lng}`;
+  request(url, function(err, res, body) {
+    if (err) {
+      console.log(err);
+    } else {
+      var street = JSON.parse(body).address.road;
+      db.findRatingEntry(street, function(err1, results1) {
+        if (err1) {
+          db.createRatingEntry(street, function(err2, results2) {
+            if (err2) {
+              console.log(err2);
+            } else {
+              console.log('created new entry');
```

Comment:
that way it is first set to green. then once you run assignRatings.js it will change the rating from green to red or yellow if its count is high enough